src/components/app/style.scss

```diff
@@ -1,3 +1,58 @@
-.app h1 {
-  color: green;
+.app {
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
```

Comment:
I don’t feel like we should go `overflow: hidden` without a good reason. For mobile we definitely want to be using the root scroller and this would make that very hard.

Reply:
Just to write down what we talked about in the office: I think we should use per-element scrollers for our compression option elements. If iOS gives us troubles, we'll work around them (potentially by reducing niceness on that platform). Basically, I'm sick of scrolling being broken on iOS.

Reply:
also trying to use fixed for the fixed portions of the UI would be a nightmare with auto-hiding address bar.